tests/test_mlmodel.py

```diff
@@ -64,6 +64,9 @@ def setup_class(cls):
         cls.manager = ResourceManager([cls.mlmodel])
         cls.service = MLModelRPCService(cls.manager)
 
+    #  ignore warning about our out-of-bound int casting (i.e. uint32 -> int16) because we don't store uint32s for int16 & uint16 tensor
```

Comment:
numpy was giving 16 warnings re: these tests because we cast uint32 to uint16 & int16

but this shouldn't be a problem as per comment